### PR TITLE
Variable config file

### PR DESCRIPTION
**Pull request by BenignBeppe**

### CODE REVIEW

**Comment by lokal-profil** [comment]
This works fine as long as there is always a `hb_config.py` file in the directory. If you rename it and give the new name as an argument the other classes fail to find it during their import (which happens before the `imp.load_source()`.

This situation happens if you e.g. want to have `config_teahouse.py` and `config_twa.py`

File: config_reader.py [hunk 1]
```diff
@@ -0,0 +1,15 @@
+import imp
+import os
+
+config = None
+
```
Comment (lokal-profil) [inline config_reader.py:9]: should be two blank lines per pep8

File: config_reader.py [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+def load_config_module(module_path):
+    module_name = os.path.splitext(os.path.basename(module_path))[0]
+    globals()["config"] = imp.load_source(module_name, module_path)
+
```
Comment (lokal-profil) [inline config_reader.py]: should be two blank lines per pep8

File: config_reader.py [hunk 3]
```diff
@@ -0,0 +1,15 @@
+import imp
```
Comment (lokal-profil) [inline config_reader.py:5]: Add a short docstring. Maybe just something like:
"mechanism for dynamically loading config file/module based on command line argument"

**Review by lokal-profil — CHANGES_REQUESTED** [review]
Only minor nitpicking. Otherwise everything looks good.

File: config_reader.py (hunk 3 above)
Comment (BenignBeppe) [reply config_reader.py:5]: Added docstrings for module and the functions.

File: config_reader.py (hunk 1 above)
Comment (BenignBeppe) [reply config_reader.py:9]: Fixed.

**Pull request merged**